packages/frontend/src/pages/Home.tsx

```diff
@@ -60,11 +62,14 @@ const Home: FC = () => {
                     />
                 ) : (
                     <LandingPanel
-                        hasSavedChart={!!savedChartStatus.data}
                         userName={user.data?.firstName}
                         projectUuid={project.data.projectUuid}
                     />
                 )}
+
+                <SpaceBrowser projectUuid={project.data.projectUuid} />
+
+                <RecentlyUpdatedPanel projectUuid={project.data.projectUuid} />
```

Comment:
Do we want to show these with the onboarding panel?

Reply:
good catch